docs/src/examples/pytorch_a2c_ppo.rst

```diff
@@ -0,0 +1,84 @@
+***********************************************
+Example with ikostrikov/pytorch-a2c-ppo-acktr
+***********************************************
+
+.. note ::
+
+    If database not setup or Oríon not installed, follow setup instructions: :doc:`../installing` and :doc:`../database`
+
+Intro
+========
+
+Here, we are looking to update the ikostrikov/pytorch-a2c-ppo-acktr Reinforcement Learning algorithm implementations to use Oríon to find the best hyperparameters while trying to prevent overfitting via a validation set of random evaluation seeds in the environment.
+
+What to change
+===============
+
+
+To get the original repository of [ikostrikov](https://github.com/ikostrikov/pytorch-a2c-ppo-acktr) to work using Orion, we make a couple of changes.
+
+First, we fork the original repo at commit hash: 4d95ec364c7303566c6a52fb0a254640e931609d
+
+To the top of
+
+.. code-block:: bash
+
+    main.py
+
+we add:
+
+.. code-block:: python
+
+    #!/usr/bin/env python
+    from orion.client import report_results
+
+
+and then we run
+
+.. code-block:: bash
+
+    chmod +x main.py
+
+To make it executable.
+
+Then, we ensure that we evaluate on a separate set of hold out random seeds for the environment (which should be different than the test set and training seed). For MuJoCo environments where the random seed has an effect, we can simply set the random seed before a rollout. In Atari, we would have to create a new validation set of rollouts perhaps with different human starts.
```

Comment:
I think we should first show how to use it with current implementation and then say that it is not a good metric to optimize because of overfitting and point to the new file you implemented. Ohterwise is sounds like orion itself requires such a change, while it's something that is lacking in general. The execution with orion does not require it, it works without it, but we do not recommand it. The file `eval.py` is required for the sake of fair benchmarking.